runtime/common/BaseRemoteRESTQPU.h

```diff
@@ -378,6 +378,7 @@ class BaseRemoteRESTQPU : public cudaq::QPU {
       func->setAttr(cudaq::entryPointAttrName, builder.getUnitAttr());
     auto moduleOp = builder.create<ModuleOp>();
     moduleOp.push_back(func.clone());
+    moduleOp->setAttrs(m_module->getAttrDictionary());
```

Comment:
Do we need to add a similar call to `runtime/common/BaseRestRemoteClient.h`: 164? e.g.,

```
moduleOp->setAttrs((*module)->getAttrDictionary());
```

Reply:
I suspect so. Without that additional change, I was still getting the same failure described in the issue. With that change, it appears to work for me.

Reply:
Very likely.